src/View/Helper/HtmlHelper.php

```diff
@@ -524,6 +541,10 @@ public function script($url, array $options = []): ?string
     public function scriptBlock(string $script, array $options = []): ?string
     {
         $options += ['block' => null];
+        $nonce = $this->_View->getRequest()->getAttribute('cspScriptNonce');
+        if (!isset($options['nonce']) && $nonce) {
+            $options['nonce'] = $nonce;
+        }
```

Comment:
```suggestion
        $options += ['block' => null, 'nonce' => $this->_View->getRequest()->getAttribute('cspScriptNonce')];
```

Wouldn't this be simpler? `formatAttributes()` skips keys with `null` values.

Reply:
Yes. this would be better. :bow: